src/main/java/nl/hsac/fitnesse/junit/allure/JUnitAllureFrameworkListener.java

```diff
@@ -272,12 +281,20 @@ private void addLabels(WikiPage page) {
         return labels;
     }
 
-    protected String[] getTags(WikiPage page) {
+    private String[] getTags(WikiPage page) {
         String[] tags = new String[0];
         String tagInfo = page.getData().getProperties().get("Suites");
         if (null != tagInfo) {
             tags = tagInfo.split(",");
         }
         return tags;
     }
+
+    private boolean reportTestPage(String pageName) {
+        if (!skipSpecialPages) {
+            return true;
+        } else {
+            return !pageName.matches(".*(\\.SuiteSetUp|\\.SuiteTearDown)$");
```

Comment:
I always prefer to create/compile regular expressions once (as `static final`) and then only use it when needed.
Is there a particular reason you make a new one here on each invocation?

Reply:
I agree, precompiled is better (and faster). Changed.